dataprofiler/tests/profilers/test_column_profile_compilers.py

```diff
@@ -262,9 +264,10 @@ def test_disabling_columns_during_primitive_diff(self):
                 },
                 "t-test": {
                     "t-statistic": -1.9674775073518591,
-                    "conservative": {"df": 1, "p-value": 0.29936264581081673},
+                    "conservative": {"df": 1.0, "p-value": 0.29936264581081673},
```

Comment:
df should be int, right?

Reply:
I just checked L561 in `numerical_column_stats.py` and it shows the below... So, I think we are good with `1.0`?

```
results["conservative"]["df"] = float(conservative_df)
```